static/app/components/events/interfaces/sourceMapsDebuggerModal.tsx

```diff
@@ -0,0 +1,1033 @@
+            <TabPanels.Item key="fetching">
+              <CheckList>
+                <ScrapingSourceFileAvailableChecklistItem
+                  facts={sourceResolutionResults}
+                />
+                <ScrapingSourceMapAvailableChecklistItem
+                  facts={sourceResolutionResults}
+                />
+              </CheckList>
+              {scrapingProgressPercent === 1 ? (
+                <ChecklistDoneNote />
+              ) : (
+                <VerifyAgainNote />
+              )}
+            </TabPanels.Item>
+          </StyledTabPanels>
+        </Tabs>
+      </Body>
+      <Footer>
+        <Link
+          to=""
+          onClick={e => {
+            e.stopPropagation();
+            openModal(modalProps => (
+              <FeedbackModal
+                featureName="sourcemaps-debugger"
+                feedbackTypes={['This was helpful', 'This was not helpful']}
+                {...modalProps}
+              />
+            ));
+          }}
+        >
+          Was this helpful? <IconMegaphone size="xs" />
+        </Link>
+      </Footer>
+    </Fragment>
+  );
+}
+
+function CheckListItem({children, title, status}: PropsWithChildren<CheckListItemProps>) {
+  return (
+    <ListItemContainer>
+      <CheckMarkContainer>
+        {
+          {
+            none: <IconCircle size="md" color="gray200" />,
+            checked: <IconCheckmark size="md" color="green300" isCircled />,
+            alert: <IconWarning size="md" color="yellow300" />,
+            question: <IconQuestion size="md" color="gray300" />,
+          }[status]
+        }
+        <Line className="source-map-debugger-modal-checklist-line" />
+      </CheckMarkContainer>
+      <ListItemContentContainer>
+        <ListItemTitleWrapper>
+          <ListItemTitle status={status}>{title}</ListItemTitle>
+        </ListItemTitleWrapper>
+        {children}
+      </ListItemContentContainer>
+    </ListItemContainer>
+  );
+}
+
+function InstalledSdkChecklistItem({facts}: {facts: SourceResoultionResults}) {
+  const itemName = 'Installed SDK supports Debug IDs';
+
+  if (facts.sdkDebugIdSupport === 'needs-upgrade') {
+    return (
+      <CheckListItem status="alert" title={itemName}>
+        <CheckListInstruction type="muted">
+          <h6>Outdated SDK</h6>
+          {facts.sdkVersion ? (
+            <p>
+              You are using version <MonoBlock>{facts.sdkVersion}</MonoBlock> of the
+              Sentry SDK which does not support debug IDs. You should upgrade to at least
+              version <MonoBlock>7.56.0</MonoBlock>.
+            </p>
+          ) : (
+            <p>
+              You are using an outdated version of the Sentry SDK which does not support
+              debug IDs. You should upgrade to at least version{' '}
+              <MonoBlock>7.56.0</MonoBlock>.
+            </p>
+          )}
+          <p>
+            If upgrading the SDK is not an option for you, you can use the "Release Name"
```

Comment:
l: should we add a link that switches tabs here? Or overkill?

Reply:
No good idea. I put that off at first because getting iteration 0 to work was more important.